go/toolchains.rst

```diff
@@ -268,7 +268,7 @@ This downloads a Go SDK for use in toolchains.
 | supports. Go versions that rules_go doesn't support may not be specified,                                  |
 | since the download SHA-256 sums are not known.                                                             |
 +--------------------------------+-----------------------------+---------------------------------------------+
-| :param:`urls`                  | :type:`string_list`         | :value:`[https://dl.google.com/go/{}`       |
+| :param:`urls`                  | :type:`string_list`         | :value:`[https://dl.google.com/go/{}]`       |
```

Comment:
Please delete a space after this so the right edge of the table lines up. The table won't render without it.

Reply:
Done